### ISSUE TITLE
Possibly inaccurate soil salinity values

### ISSUE THREAD

**Issue by sarahbayley** [treflehq/trefle-api#90]
Not exactly a 'bug', but a data issue I hope can be addressed or clarified. I'm noticing that the soil salinity ratings don't match very well with the USDA source and I can't seem to find where else that data is sourced from.

The USDA uses the following criteria for their rating:

```
What is the plant’s tolerance to soil salinity? Tolerance to a soil salinity level is defined as only a slight reduction (not greater than 10%) in plant growth. None = tolerant to a soil with an electrical conductivity of the soil solution extract of 0-2 dS/m; Low = tolerant to 2.1-4.0 dS/m; Medium = tolerant to 4.1-8.0 dS/m; High = tolerant to greater than 8.0 dS/m.

    None, Low, Medium, High
```

Trefle, on the other hand, uses a scale of 0 (untolerant) to 10 (hypersaline).

I'm wondering what the criteria is for placement on this 0-10 scale and which source is using this scale. This will help me label/explain these values in my app or report any errors if there are any.

Here's a sampling of plants with salinity ratings above 1 and their corresponding rating on the USDA site:

Trefle ID | Scientific Name | Trefle soil salinity | USDA soil salinity | USDA source link

107095 | Argentina anserina | 2 | low | [Link](https://plants.usda.gov/java/charProfile?symbol=ARAN7)  
149757 | Leymus arenarius | 3 | high | [Link](https://plants.usda.gov/java/charProfile?symbol=LEAR11) 
189317 | Trifolium fragiferum | 3 | high | [Link](https://plants.usda.gov/java/charProfile?symbol=TRFR2) 
175834 | Rosa rugosa | 3 | low | [Link](https://plants.usda.gov/java/charProfile?symbol=RORU) 
147570 | Lathyrus japonicus | 5 | medium | [Link](https://plants.usda.gov/java/charProfile?symbol=LAJA) 
107435 | Armeria maritima | 6 | low | [Link](https://plants.usda.gov/java/charProfile?symbol=ARMA6) 
111034 | Bassia hyssopifolia | 8 | high | [Link](https://plants.usda.gov/java/charProfile?symbol=BAHY) 
177013 | Ruppia maritima | 8 | medium | [Link](https://plants.usda.gov/java/charProfile?symbol=RUMA5) 
131629 | Eleocharis parvula | 8 | medium | [Link](https://plants.usda.gov/java/charProfile?symbol=ELPA5)

**Comment by lambda2** [comment]
Good catch, and sorry for the very late answer — this deserves a proper explanation, because the values you compared are not on the same scale at all.

**Where the 0–10 scale comes from.** Trefle's `soil_salinity` is *not* derived from the USDA None/Low/Medium/High tolerance classes. It comes from the French Baseflor/Catminat dataset (Philippe Julve), which uses Ellenberg-style ecological indicator values: the 0–10 figure describes where the species' natural habitats sit along a salinity gradient (0 = glycophyte habitats → 9–10 = hypersaline), not how much growth reduction the plant tolerates in cultivation. The USDA rating is an agronomic tolerance class defined by electrical conductivity thresholds. Both can be "correct" for the same species while disagreeing numerically — *Armeria maritima* is a good example: a coastal-habitat indicator scores it high, while USDA rates its cultivated tolerance low. So there is no 1:1 mapping between the two, and comparing them will always look inconsistent.

For labelling in your app: treat Trefle's value as "habitat salinity indicator (Ellenberg-like, Baseflor)" rather than "salinity tolerance". The docs will be updated to say this explicitly (same caveat applies to `light`, `atmospheric_humidity`, `ground_humidity` and `soil_nutriments`, which come from the same indicator system and are only calibrated for the European temperate flora).

**What changed since.** #250 added per-field provenance: every value ingested from now on is recorded as a fact with its source, and `GET /api/v1/species/{id}/facts` exposes that trail, so "which source is using this scale" becomes answerable from the API itself. Values are also validated against plausible ranges at ingestion (`config/traits.yml`), and conflicting claims between sources are kept visible instead of silently overwritten.

Keeping this open until the field documentation on docs.trefle.io spells out the scale's origin.
